Letterbook.IntegrationTests/PostsTests.cs

```diff
@@ -90,6 +126,26 @@ public async Task CanDraftNote()
 		Assert.NotNull(body.Id);
 		Assert.NotEqual(Uuid7.Empty, body.Id);
 		Assert.Equal(dto.Contents.First(), body.Contents.FirstOrDefault(), ContentComparer);
+
+		if(dto.Audience.Count > 0 || dto.AddressedTo.Count > 0)
+		{
+			const string expected = "urn:message:Letterbook.Workers.Contracts:ActivityMessage send";
+			Assert.Contains(expected, _host.Spans.Where(a => a.TraceId == traceId).Select(s => s.DisplayName));
```

Comment:
This is probably worth calling out explicitly. `_host.Spans` is the in-memory observable collection of spans produced by the application. This lets us use them in our tests, which makes it relatively easy to assert that things happened in the queue workers. This would have otherwise been nearly impossible.